pyrogram/client/types/messages_and_media/message.py

```diff
@@ -2684,12 +2687,10 @@ def forward(
         """
         if as_copy:
             if self.service:
-                raise ValueError("Unable to copy service messages")
-
-            if self.game and not self._client.is_bot:
-                raise ValueError("Users cannot send messages with Game media type")
-
-            if self.text:
+                log.warning("Unable to copy service messages, message_id: {}".format(self.message_id))
```

Comment:
Maybe also mention the source chat id (`self.chat.id`) and not only the message id.